content/help/faq/distributed-tracing/how-to-support-tracing.md

```diff
@@ -0,0 +1,19 @@
+---
+title: What is required for distributed tracing with Istio?
+weight: 10
+---
+
+Istio enables reporting of trace spans for workload-to-workload communications within a mesh. However, in order for various trace spans to be stitched together for a complete view of the traffic flow, applications must propagate the trace context between incoming and outgoing requests.
+
+In particular, Istio relies on applications to [propagate the B3 trace headers](https://github.com/openzipkin/b3-propagation), as well as the Envoy-generated request ID. The complete set is:
+
+- `x-request-id`
```

Comment:
I think this should be broken out more, especially noting that `x-request-id` and `x-ot-span-context` are envoy specific. LightStep's `x-ot-span-context`  header is not required in other istio setups as far as I know. 

Put another way, only in envoy would using a commercial setup like datadog still require sending their competitor lightstep's proprietary header.

Reply:
Broke out the LightStep header as requested.